CONTRIBUTING.md

```diff
@@ -79,7 +79,12 @@ section.
   * Make sure it has the `license` and `images` keys.
   * Make sure the text `,Odoo Community Association (OCA)` is appended
     to the `author` text.
-* Don't use your company logo or your corporate branding. Using the website, the author and the list of contributors is enough for people to know about your employer/company and contact you.
+  * The `website` key must be `https://github.com/OCA/<repo>`,
```

Comment:
Again, I myself prefer just the repo as it is the landing page of the whole project. So 'more relevant' is a matter of opinion likewise.